pixiedust/display/chart/renderers/mapbox/templates/mapView.html

```diff
@@ -51,7 +51,7 @@
     mapboxgl.accessToken="{{this.options.get("mapboxtoken")}}";
     var map = new mapboxgl.Map({
         container: "map{{prefix}}",
-        style: "mapbox://styles/mapbox/light-v9"
+        style: "mapbox://styles/mapbox/{{this.options.get("basemap")}}"
```

Comment:
@rajrsingh i would suggest providing a default value just in case`basemap` is not found: `this.options.get("basemap", "light-v9")`